src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -247,6 +259,19 @@ public Owner updateOwnerContentAccess(Owner owner, String updatedList, String up
                 this.contentAccessCertificateCurator.deleteForOwner(owner);
             }
 
+            // Revoke entitlements and remove activation key pools if moving from entitlement mode to SCA mode
+            if (this.isTransition(currentMode, updatedMode, ContentAccessMode.ENTITLEMENT,
+                ContentAccessMode.ORG_ENVIRONMENT)) {
+                try {
+                    RevokeEntitlementsJobConfig jobConfig = new RevokeEntitlementsJobConfig();
+                    jobConfig.setJobArgument(RevokeEntitlementsJob.OWNER_KEY, owner.getKey());
+                    jobManager.queueJob(jobConfig);
```

Comment:
I'm not sure how to feel about the job coming into existence here but we have no means of tracking its state. I don't know that we have a more reliable way to do it, and even at the owner update endpoint, we can't return the job status without omitting the owner output, so this may be our lot anyway.

Are we okay with the job blinking into and out of existence transparently like this? @nikosmoum